scripts/add_field_for_stuck_registrations/README.md

```diff
@@ -0,0 +1,11 @@
+Add Field For Contacts Stuck In The Registration Consent State
+----------------------
+The add_field_to_stuck_registration_contacts.py script takes in a CSV in stdin,
+and outputs a CSV on stdout. The input CSV is contacts in the stuck state
```

Comment:
It doesn't seem to output anything on stdout

Reply:
Also, we should probably rather mention `The input CSV is the UUID's of contacts in the stuck state`, so that it's clear what the content of the input should be.

Reply:
I forgot to make changes to the readme, thanks for pointing this out